sample/src/main/java/ng/max/slideview/sample/MainActivity.java

```diff
@@ -22,5 +22,23 @@ public void onSlideComplete(SlideView slideView) {
                 slideView.setText("Disabled");
             }
         });
+        ((SlideView) findViewById(R.id.slider6)).setOnSlideCompleteListener(new SlideView.OnSlideCompleteListener() {
+            @Override
+            public void onSlideComplete(SlideView slideView) {
+               if (slideView.getReverseSlide()) {
+                   slideView.setReverseSlide(false);
+                   slideView.setText("Accept");
+                   slideView.setButtonBackgroundColor(getColor(R.color.colorAccent));
```

Comment:
The `getColor()` method used in lines 31-39  require API level 23 but the project's minimum SDK is 14. Consider using `ContextCompat.getColor()` instead.